### ISSUE TITLE
TypeError: Cannot read property 'result' of undefined

### ISSUE THREAD

**Issue by TehShrike** [jonathantneal/postcss-advanced-variables#49]
I ran into this crash while using this plugin to transform the styles inside of a [Svelte](https://svelte.technology/) component.

If no source file path is given, and `@import` is used in the template, this TypeError is thrown:

```
TypeError: Cannot read property 'result' of undefined
    at getImportOpts (/some-path/node_modules/postcss-advanced-variables/index.bundle.js:451:79)
    at transformImportAtrule (/some-path/node_modules/postcss-advanced-variables/index.bundle.js:377:24)
    at /some-path/node_modules/postcss-advanced-variables/index.bundle.js:683:5
    at Array.forEach (<anonymous>)
    at transformNode (/some-path/node_modules/postcss-advanced-variables/index.bundle.js:667:22)
    at /some-path/node_modules/postcss-advanced-variables/index.bundle.js:739:3
    at /some-path/node_modules/precss/index.bundle.js:31:12
    at <anonymous>
```

Test case/fix coming momentarily.

**Comment by jonathantneal** [comment]
Published in [v2.3.2](https://github.com/jonathantneal/postcss-advanced-variables/blob/2.3.2/CHANGELOG.md)! Thank you, again.

**Comment by TehShrike** [comment]
My pleasure!  Thanks for the quick release.
